dcs/terrain/falklands.py

```diff
@@ -0,0 +1,668 @@
+        self.parking_slots.append(ParkingSlot(
+                crossroad_idx=48, position=mapping.Point(-146759.46875, -778685.9375, self._terrain), large=False, heli=False,
+                airplanes=True, slot_name='HAS_11', length=21.0, width=15.0, height=8.0, shelter=False))
+        self.parking_slots.append(ParkingSlot(
+                crossroad_idx=49, position=mapping.Point(-146762.90625, -778666.5, self._terrain), large=False, heli=False,
+                airplanes=True, slot_name='HAS_12', length=21.0, width=15.0, height=8.0, shelter=False))
+        self.parking_slots.append(ParkingSlot(
+                crossroad_idx=50, position=mapping.Point(-146745.671875, -778764.4375, self._terrain), large=False, heli=False,
+                airplanes=True, slot_name='HAS_07', length=21.0, width=15.0, height=8.0, shelter=False))
+        self.parking_slots.append(ParkingSlot(
+                crossroad_idx=51, position=mapping.Point(-146749.109375, -778745, self._terrain), large=False, heli=False,
+                airplanes=True, slot_name='HAS_08', length=21.0, width=15.0, height=8.0, shelter=False))
+        self.parking_slots.append(ParkingSlot(
+                crossroad_idx=52, position=mapping.Point(-146752.578125, -778725.375, self._terrain), large=False, heli=False,
+                airplanes=True, slot_name='HAS_09', length=21.0, width=15.0, height=8.0, shelter=False))
+        self.parking_slots.append(ParkingSlot(
+                crossroad_idx=53, position=mapping.Point(-146756.046875, -778705.75, self._terrain), large=False, heli=False,
+                airplanes=True, slot_name='HAS_10', length=21.0, width=15.0, height=8.0, shelter=False))
+        self.parking_slots.append(ParkingSlot(
+                crossroad_idx=54, position=mapping.Point(-146742.84375, -778458.3125, self._terrain), large=False, heli=False,
+                airplanes=True, slot_name='R01', length=21.0, width=15.0, height=8.0, shelter=False))
+        self.parking_slots.append(ParkingSlot(
+                crossroad_idx=55, position=mapping.Point(-147472.625, -777959.875, self._terrain), large=False, heli=False,
+                airplanes=True, slot_name='F8', length=26.0, width=24.0, height=11.0, shelter=False))
+
+# The following section belongs in the Terrain subclass, not in the Airport
+
+class Falklands(Terrain):
+    center = {"lat": 52.468, "long": 59.173}
+    bounds = mapping.Rectangle(74967, -114995, -129982, 129991)
+    map_view_default = MapView(mapping.Point(0, 0), 1000000)
+    city_graph = None
+    temperature = [
+        (5, 17),
+        (5, 17),
+        (2, 14),
+        (2, 14),
+        (2, 14,
+        (-5, 11),
+        (-5, 11),
+        (-5, 11),
+        (1, 15),
+        (1, 15),
+        (1, 15),
+        (5, 17)
+    ]
+    assert (len(temperature) == 12)
+
+    def __init__(self):
+        super(Falklands, self).__init__("falklands")

```

Comment:
Did the tool generate this? This doesn't match the constructor for `Terrain` any more. See the other terrains for examples. One of the things you'll notice is that you need projection parameters so we can do lat/on <-> x/y conversions. https://github.com/pydcs/dcs/blob/master/tools/export_map_projection.py is the tool to do that. See the docs at the top.